### ISSUE TITLE
Error in firebase.mlkit.useCustomModel: TypeError: Cannot read property 'trim' of undefined

### ISSUE THREAD

**Issue by mesirendon** [EddyVerbruggen/nativescript-plugin-firebase#1427]
Hi. I'm trying to get the MLKit reading my model and the labels for it.

I've tried with the Vue Component, but it doesn't allow to load the component. Then I loaded the still image using the camera. Here's the method that wants to apply the custom model:

```javascript
import { MLKitCustomModelResult, useCustomModel } from "nativescript-plugin-firebase/mlkit/custommodel";

useCustomModel({
      image: this.image,
      localModelFile: '~/custommodel/rust.tflite',
      labelsFile: '~/custommodel/rust_labels.txt',
      modelInput: [{
        shape: [1, 299, 299, 3],
        type: 'QUANT',
      }],
    })
      .then((result: MLKitCustomModelResult) => console.log(JSON.stringify(result.result)))
      .catch(console.error)
```

I'm always getting this: 

```
JS: 'Error in firebase.mlkit.useCustomModel: TypeError: Cannot read property \'trim\' of undefined'
JS: TypeError: Cannot read property 'trim' of undefined
```

I think it is not able to read the file contents. I also checked in the demo-ng, for still images as well for the cam feed, but it seems to happens also for the

**Comment by EddyVerbruggen** [comment]
Please share your repo.

**Comment by mesirendon** [comment]
Hi... thanks for the quick response.

Here is: https://github.com/Plants-Disease-UN-App/desplanta/blob/develop/app/components/camera/index.vue

It's worthy to mention that I've been trying also the tslite model and txt file from the demo-ng project just to check if my model and category file has the problem.

**Comment by mesirendon** [comment]
I don't know if this helps, but I see the file exists using the `tns-core-module` filesystem to check that. But the `readTextSync` method throws nothing but an empty string.

**Comment by EddyVerbruggen** [comment]
Can you change that path to `~/app/custommodel/...` and see if that helps?

**Comment by mesirendon** [comment]
Nope... same error:

```
JS: 'Error in firebase.mlkit.useCustomModel: TypeError: Cannot read property \'trim\' of undefined'
JS: TypeError: Cannot read property 'trim' of undefined
```

**Comment by EddyVerbruggen** [comment]
Perhaps those files don’t end up in your webpacked bundle so they have to be added to the copy section of the webpack config file. I’m not behind a computer, so plz check the config in demo-ng for details.

**Comment by mesirendon** [comment]
Well... I've checked, but it is actually packed. I took a screenshot.

![image](https://user-images.githubusercontent.com/1184373/66520112-2c4d8e00-eaae-11e9-9cb0-17b6774e8e23.png)

**Comment by mesirendon** [comment]
Hey Eddy... I took a look on my repo. I actually was not copying correctly the tflite file nor the txt file. I included both in the webpack config and at that point it didn't work. I deleted the platforms and node_modules folders and restarted again. After that it recognized the model and I could use my own model.

I guess it'd worth including that instruction in the repo.

Anyways... thank you very much for your help.
